qutebrowser/browser/commands.py

```diff
@@ -643,14 +643,32 @@ def scroll_perc(self, perc: {'type': float}=None,
 
     @cmdutils.register(instance='command-dispatcher', hide=True,
                        scope='window', count='count')
-    def scroll_page(self, x: {'type': float}, y: {'type': float}, count=1):
+    def scroll_page(self, x: {'type': float}, y: {'type': float}, *,
+                    top_navigate: {'type': ('prev', 'decrement')}=None,
+                    bottom_navigate: {'type': ('next', 'increment')}=None,
+                    count=1):
         """Scroll the frame page-wise.
 
         Args:
             x: How many pages to scroll to the right.
             y: How many pages to scroll down.
+            bottom_navigate: :navigate to the next page on bottom
+            top_navigate: :navigate to the previous page on top
             count: multiplier
         """
+        frame = self._current_widget().page().currentFrame()
+        if not frame.url().isValid():  # Issue 701
+            return
```

Comment:
Fair enough, let's keep it as-is then. Also I'd like your opinion about the comment above about the documentation - after that's done I'll test and merge this.